common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -226,42 +221,6 @@ private static void tryLoadItemGuesses() {
         // Check for success
     }
 
-    private static void tryLoadItemMaterialIdMap() {
```

Comment:
How do we get the data if you remove the json download? Not sure I'm following the changes here.

Reply:
I made scyu update athena so the new ids are added there, and will be always up-to-date.

Reply:
So prior to this fix we got info from two sources, old ids from Athena and new from the json file? And now all are merged into one in Athena?

Reply:
Yes